src/api/firebase.js

```diff
@@ -42,10 +60,10 @@ export function getItemData(snapshot) {
 
 		return data;
 	});
-
+	// console.log(arrayFromFirestore,'arrayFromFirestore')
```

Comment:
I think this commented out console log can be removed
```suggestion
```

Reply:
Cleaned up in my recent commit, thanks for the catch.